### PR TITLE
Bug fix GitHub images issue #44

### PR DESCRIPTION
**Pull request by sadath2001**
Hi @AlfiyaSiddique 

Issue: This is regarding #44 .

I have made the changes as follows:

https://github.com/user-attachments/assets/955043ce-6717-4cd9-94ea-bba4903ca929

Kindly let me know if any changes needed

### CODE REVIEW

File: backend/Controllers/RecipeController.js [hunk 1]
```diff
@@ -182,15 +183,84 @@ const updateRecipe = async (req, res)=>{
...
+const deleteRecipe = async (req, res) => {
+  try {
+    const recipe = await Recipe.findById(req.body.id);
+
+    if (!recipe) {
+      return res.status(404).json({ success: false, message: "Recipe not found" });
+    }
+
+    const imageName = recipe.image;
+    const owner = 'AlfiyaSiddique';
```
Comment (AlfiyaSiddique) [inline backend/Controllers/RecipeController.js:195]: @sadath2001 one improvement:
Let's store the owner, repo and branch name in env.
Also add dummy environment variable .env file with just example values. write tho short comment why this repo and owner variable required in .env file.
After making the variable can you change the hardcoded values to env values wherever name. repo and branch is used?

Comment (sadath2001) [reply backend/Controllers/RecipeController.js:195]: hi @AlfiyaSiddique , sure thing, i can do that.
let me make the change and get back to you

**Review by sadath2001 — COMMENTED** [review]
made the changes accordingly

**Comment by sadath2001** [comment]
hi @AlfiyaSiddique , i have made the changes

**Comment by AlfiyaSiddique** [comment]
@sadath2001 put .env file inside backend folder, please fix this I will merge right now

**Review by sadath2001 — COMMENTED** [review]
hi @AlfiyaSiddique , Done

**Review by AlfiyaSiddique — APPROVED** [review]

**Review by AlfiyaSiddique — APPROVED** [review]

**Comment by AlfiyaSiddique** [comment]
@sadath2001 getting this error in backend for the current change, any idea what could be the issue:
```
GET /repos/AlfiyaSiddique/ImageDatabase/contents/TastyTrails%2FRecipe%2Fec93profile%2520picture.jpeg - 404 with id A32B:2C3D00:531F7C6:53C6958:67051A44 in 262ms
Error fetching file content: {
  message: 'Not Found',
  documentation_url: 'https://docs.github.com/rest/repos/contents#get-repository-content',
  status: '404'
}
```

**Comment by AlfiyaSiddique** [comment]
Also in a new issue can you make the change if the image is not deleted due to such errors than the whole recipe data should not delete as well and give the alert something wen wrong cannot delete the recipe

**Comment by AlfiyaSiddique** [comment]
@sadath2001 any comment on above ?

**Comment by sadath2001** [comment]
hi @AlfiyaSiddique , #135 , could you let me know your steps which you performed?

i see the deletion being successful for me.
The error you are saying might be because the file you are trying to fetch doesnt exist?

Ill check by pulling the latest repo once

**Comment by sadath2001** [comment]
Hi @AlfiyaSiddique , i figured it out, while substituting the env variables, there was a slight mistake by replacing ` with ' which led to this. i apologize, ill fix it right away.

**Pull request merged**